exams/views.py

```diff
@@ -10,3 +18,27 @@ class PearsonCallbackRedirectView(RedirectView):
     """
     def get_redirect_url(self, status):  # pylint: disable=arguments-differ
         return "/dashboard?exam={status}".format(status=status)
+
+
+class PearsonSSO(APIView):
+    """
+    Views for the Pearson SSO API
+    """
+    authentication_classes = (
+        authentication.SessionAuthentication,
+        authentication.TokenAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-self-use
+        """
+        handle the post request
+        """
+        # import ipdb
+        # ipdb.set_trace()
+        student_id = request.data.get('student_id')
+        timestamp = request.data.get('timestamp')
+        session_timeout = request.data.get('session_timeout')
+
+        digest = sso_digest(student_id, timestamp, session_timeout)
+        return Response(digest)
```

Comment:
This needs to return a dictionary of `timestamp`, `session_timeout`, and `digest` so you can use that in the form.